.gitignore

```diff
@@ -78,3 +78,8 @@ docs/source/*
 .DS_Store
 
 core/_version.py
+
+# Ignore xml and json files in the root
+# Since these are generally feed downloads that we wish to ignore
+/*.json
+/*.xml
```

Comment:
This came in incidentially as part of this PR. It is really useful though, as I tend to end up with the root directory full of various feed downloads. So I left it in here.